### ISSUE TITLE
🚀 [Feature]  Group Link improvment

### ISSUE THREAD

**Issue by hermann-san** [Ocelot-Social-Community/Ocelot-Social#7540]
Hi, 
I would find it more intuitive and easier to find, if the group name Link is (also?) at the top of the page.  #5501 

![grafik](https://github.com/Ocelot-Social-Community/Ocelot-Social/assets/7679654/1bdd1466-25c1-4193-8044-f86a12ee84af)

**Comment by Abhinav28bit** [comment]
Hi , can i take this ?

**Comment by Tirokk** [comment]
Hey dear @Abhinav28bit I clarify because this issue was in a list of passible issue we just gave to an other contributor.

Okay?

**Comment by Tirokk** [comment]
Yes you are welcome to work on it @Abhinav28bit  🙏🏼

Great that you want to take on this task! 🤗
I assigned you to this issue.

I took care of your invitation to join our community as a volunteer via an e-mail invitation that you can accept.

Our conversation runs over our discord server:
https://discord.gg/AJSX9DCSUA
Please use our chat channel to ping me ( @wolle (Hamburg)#8013 ) or another mentor.

!!!
Please don’t compare from your fork to this repository in case, because the publish action will fail, because of permission problems with the credentials.
Please do not compare from your fork to this repository, as the CI will fail due to authorization issues with the credentials.
Simply push your branch directly into our repo and compare it with the master.
If you accept the invitation you have the right to do so. 😉

**Comment by Tirokk** [comment]
Please let us know here if you work on it. @Abhinav28bit

**Comment by Abhinav28bit** [comment]
yes i will start working on it by tomorrow thank you so much for giving me
this opportunity

**Comment by Tirokk** [comment]
Hey @Abhinav28bit ,
can I help you in anyway?
